src/pages/MyCoursePage.module.scss

```diff
@@ -0,0 +1,144 @@
+@import '../colors.module.scss';
+
+@mixin base-flex-col {
+    display: flex;
+    flex-direction: column;
+}
+
+@mixin flex-col {
+    @include base-flex-col;
+    justify-content: space-between;
+    align-items: center;
+}
+
+@mixin flex-row {
+    display: flex;
+    flex-direction: row;
+}
+
+@mixin card-overwrite {
+    box-shadow: none;
+    border: none;
+    border-inline-end: none !important;
+}
+
+.cardOverwrite {
+    @include card-overwrite;
+}
+
+.flexCol {
+    @include base-flex-col;
+}
+
+.flexRow {
+    @include flex-row;
+    align-items: center;
+    justify-content: space-between;
+    gap: 2%;
+}
+
+.flexRowStart {
+    @include flex-row;
+    align-items: center;
+    justify-content: flex-start;
+    gap: 5%;
+}
+
+.myCourseMenu {
+    width: 30%;
+}
+
+.outlined {
+    border-radius: 5px;
+    background-color: $colorSecondary;
+    padding: 10px;
+    margin: 5px;
+}
+
+@mixin base-background {
```

Comment:
Why are you using css mixins? Normally you can just apply multiple classes to an element or create a general sccs module where you put shared classes.

Reply:
Good question, I just remembered using them in the angular blog project. But there's no real reason.